src/game/gamearea.ts

```diff
@@ -65,6 +104,8 @@ export default class GameArea {
     // ...and add the correct one
     if (mode === Mode.MAPEDITOR) {
       this.wrapper.appendChild(this.mapEditorCanvas);
+    } else if (mode === Mode.SPLASH) {
```

Comment:
I assume we want the splash screen to appear any time before we open the map editor or start running a match?